keyboards/handwired/ndk/ndk16/keymaps/default/keymap.c

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2023 kevin (@itskevin-zz)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*
+     * ┌───┬───┬───┬───┬───┐     
+     * │ e │NUM│ / │ * │ - │
+     * ├───┼───┼───┼───┼───┤
+     * │   │ 7 │ 8 │ 9 │   │
+     * │   ├───┼───┼───┤ + │
+     * │   │ 4 │ 5 │ 6 │   │
+     * │   ├───┼───┼───┼───┤
+     * │   │ 1 │ 2 │ 3 │   │
+     * ├───┼───┴───┼───┤ ← │
+     * │ D │   0   │ . │   │
+     * └───┴───────┴───┴───┘
+     */
+    [0] = LAYOUT_numpad_5x5(
+        KC_MUTE, KC_NUM, KC_PSLS, KC_PAST,  KC_PMNS,
+        KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_P4,   KC_P5,   KC_P6,
```

Comment:
This should follow the other numpad layouts.  Eg: 

```suggestion
        KC_P7,   KC_P8,   KC_P9,
        KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
```

Reply:
updated the matrix and numpad layout.